scopes/semantics/schema/schema.task.ts

```diff
@@ -33,6 +33,8 @@ export class SchemaTask implements BuildTask {
     const rootDir = context.capsuleNetwork.capsulesRootDir;
     await pMapSeries(capsules, async (capsule) => {
       const component = capsule.component;
+      const isSchemaTaskDisabled = component.state.aspects.get(this.aspectId)?.data?.disabled;
```

Comment:
Better to have an api in the schema main runtime that calculate that data then call it from the task

Reply:
@luvkapur , I guess what @GiladShoham  wants is a method `getData()` in `schema.main` which returns `component.state.aspects.get(this.aspectId)?.data`

Reply:
@davidfirst @GiladShoham I added 
```
 isSchemaTaskDisabled(component: Component) {
    return component.state.aspects.get(SchemaAspect.id)?.data?.disabled;
  }
  ```
  to `schema.main`
  
  and invoke it from the task like this
  `const isTaskDisabled = this.schema.isSchemaTaskDisabled(component);`